### DISCUSSION TITLE
Symbologies

### DISCUSSION THREAD

**Discussion by abubelinha** [bionomia/bionomia#353] [Ideas] [answered]
Just want to drive attention about this:

https://ux.stackexchange.com/questions/10911/what-would-be-a-good-non-religious-icon-to-represent-that-a-person-has-died

Nothing that I am worried about, but some people might be.

**Comment by dshorthouse** [comment, marked as answer]
Removed &#42; and &dagger; from lifespan with https://github.com/bionomia/bionomia/commit/89c3faad8a82755c12f7e612798c8168679c6a17
